src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -234,7 +234,7 @@ export const View = (props: Props) => {
                 }).href
               }
             >
-              {l10n.getString("exposure-card-cta")}
+              {l10n.getString("exposure-card-resolve-exposures-cta")}
```

Comment:
After resolving an exposure we need to show [this indicator](https://www.figma.com/design/DTbmXzCQCw2PxXpHQW8yfW/Monitor-MVP-Enhancements?node-id=3809-6687&node-type=instance&m=dev) on the respective exposure card.